crates/uv/src/commands/pip_check.rs

```diff
@@ -42,18 +46,50 @@ pub(crate) fn pip_check(
 
     // Build the installed index.
     let site_packages = SitePackages::from_executable(&venv)?;
+    let packages: Vec<&InstalledDist> = site_packages.iter().collect();
 
-    let mut is_compatible = true;
-    // This loop is entered if and only if there is at least one conflict.
-    for diagnostic in site_packages.diagnostics()? {
-        is_compatible = false;
-        writeln!(printer.stdout(), "{}", diagnostic.message())?;
-    }
+    let s = if packages.len() == 1 { "" } else { "s" };
+    writeln!(
+        printer.stderr(),
+        "{}",
+        format!(
+            "Checked {} in {}",
```

Comment:
Or "Audited", up to you.

Reply:
Hm does it make sense to match our other use of "Audited" here? It might... but I feel like this context is different. I choose this message before I added a second saying all of the packages are compatible.